packages/relay-service/lib/index.js

```diff
@@ -69,12 +70,14 @@ const getMessages = async ({keyPairDocs, limit = 20}) => {
       )}&payload=${toBase64(payload)}`,
     );
 
+    const data = result.data;
+
     const messages = await Promise.all(
-      result.data.map(async item => {
+      data.map(async item => {
         const keyPairDoc = keyPairDocs.find(doc => doc.controller === item.to);
         assert(!!keyPairDoc, `keyPairDoc not found for did ${item.to}`);
         const keyAgreementKey = await getDerivedAgreementKey(keyPairDoc);
-        const jwe = JSON.parse(item.msg);
+        const jwe = fromBase64(item.msg);
```

Comment:
you cant assume its always base64

Reply:
I've updated the impl, it will now check if the value is base64 before converting.